core/templates/dev/head/pages/exploration_editor/translation_tab/translation_tab_directive.html

```diff
@@ -1,4 +1,4 @@
-<div class="oppia-translation-tab">
+<div class="oppia-translation-tab" ng-joy-ride="translationTutorial" config="TRANSLATION_TUTORIAL_OPTIONS" on-finish="onFinishTutorial()" on-skip="onSkipTutorial()">
```

Comment:
Can we make it more readable like [this](https://github.com/oppia/oppia/blob/8db6c74e3a4ae19ab216c584cb726492db5c901f/core/templates/dev/head/pages/exploration_editor/translation_tab/state_translation_directive.html#L52)?

Reply:
Done.